metadata-ingestion/src/datahub/utilities/mapping.py

```diff
@@ -232,8 +232,8 @@ def get_match(self, match_clause: Any, raw_props_value: Any) -> Optional[Match]:
         if type(raw_props_value) not in Constants.OPERAND_DATATYPE_SUPPORTED or type(
             raw_props_value
         ) != type(match_clause):
-            return None
+            return False
```

Comment:
This is returning `bool` instead of `Optional[Match]`. This changes the contract of this method. You marked my earlier comment as resolved. Please explain why are you changing the contract here?